### PR TITLE
Issue model

### PR DESCRIPTION
**Pull request by teabagk7**
Your suggestions etc.

### CODE REVIEW

**Comment by dsavenko** [comment]
Some things are not clear:

1. How the dates are stored (the date of opening and closing, as well as the dates of all changes to the issue);

2. How the user information is stored (who created/change/resolved the issue);

3. How to get a list of all issues based on status (all/opened/resolved);

**Review by dsavenko — CHANGES_REQUESTED** [review]
Some things are not clear:
1. How the dates are stored (the date of opening and closing, as well as the dates of all changes to the issue);
2. How the user information is stored (who created/change/resolved the issue);
3. How to get a list of all issues based on status (all/opened/resolved);

Please, clarify in the document.

**Comment by teabagk7** [comment]
[Изучить модель данных Git. Разработать план как реализовать хранение тикетов в Git'e в терминах этой модели.](https://trello.com/c/JBb8jvtM/1-git-git-e)

**Review by dsavenko — APPROVED** [review]

File: Image_of_issue.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+##### 3.blob description:
+В blob объекте *description* хранится текст содержащий текущий description issue. При создании issue присваивается весь текст полученный от пользователя при вводе начиная со второй строки.
+
+Из этих blob объектов создается дерево. Далее создается коммит на дерево с blob объектами, который будет отражать дату создания и автора issue. Хэш этого коммита (далее обозначается как хэш issue) выдается пользователю по завершению работы команды, как хэш по которому пользователь может осуществлять операции над issue.
+Создается новый blob с названием равным хэшу issue, содержащий в себе строку с хэшем issue. (В пункте *“изменение issue”* обосновываются причины создания этого blob объекта). Из этого blob объекта создается дерево. Проверяется наличие ссылки *".issues-reserved"* в репозитории, делается коммит на созданное дерево, если ссылка существует, то в parent нового коммита указывается коммит на который указывает ссылка, в обратном случае коммит остается без parent, и создается ссылка *".issues-reserved"* указывающая на этот коммит.
```
Comment (akhismat) [inline Image_of_issue.md]: Имя рефа "ISSUES", обнови пожалуйста

**Comment by akhismat** [comment]
Предлагаю переименовать Image_of_issue.md в MODEL.md

**Review by kevromster — APPROVED** [review]

**Review by makrushin-evgenii — APPROVED** [review]

**Review by akhismat — COMMENTED** [review]
Я имела ввиду переимоновать файл в MODEL.md, а не заголовок пулл реквеста:)

**Review by MaxIvanov1997 — APPROVED** [review]

**Review by Tatiotaeryy — APPROVED** [review]

**Review by akhismat — APPROVED** [review]

**Pull request merged**